collect_app/src/test/java/org/odk/collect/android/activities/viewmodels/SplashScreenViewModelTest.kt

```diff
@@ -8,21 +8,18 @@ import org.mockito.Mockito.`when`
 import org.mockito.Mockito.mock
 import org.odk.collect.android.preferences.keys.ProjectKeys
 import org.odk.collect.android.preferences.source.SharedPreferencesSettings
-import org.odk.collect.android.utilities.LaunchState
 import org.odk.collect.projects.Project
 import org.odk.collect.projects.ProjectsRepository
 
 class SplashScreenViewModelTest {
 
     private lateinit var generalSettings: SharedPreferencesSettings
-    private lateinit var launchState: LaunchState
```

Comment:
Not sure why this was here. Probably in use at some point.

Reply:
probably